public/includes/find-house.php

```diff
@@ -0,0 +1,65 @@
+<?php
+require_once('db.php');
+
+function get_states() {
+    $mysqli = db_connection();
+    $sqlQuery = "SELECT * FROM `states` WHERE 1";
```

Comment:
I think `WHERE 1` is not necessary in this query.
```suggestion
    $sqlQuery = "SELECT * FROM `states`";
```